components/vc/pdu/include/HW/HW_adc_componentSpecific.h

```diff
@@ -24,12 +24,24 @@
 
 typedef enum
 {
+    ADC_BANK1_CHANNEL_MUX_LP1_SNS,
+    ADC_BANK1_CHANNEL_MUX_LP2_SNS,
+    ADC_BANK1_CHANNEL_MUX_LP3_SNS,
+    ADC_BANK1_CHANNEL_MUX_LP4_SNS,
+    ADC_BANK1_CHANNEL_UVL_BATT,

```

Comment:
I would put the battery voltage on at the same time as hp cs and not thermisors. We have dual simultaneous sampling setup so the measurements at the same time will give you a better seasurement of live power draw off the heaviest loads